models/mail.go

```diff
@@ -160,7 +161,12 @@ func composeIssueMessage(issue *Issue, doer *User, tplName base.TplName, tos []s
 	if err != nil {
 		log.Error(3, "HTMLString (%s): %v", tplName, err)
 	}
-	msg := mailer.NewMessageFrom(tos, fmt.Sprintf(`"%s" <%s>`, doer.DisplayName(), setting.MailService.User), subject, content)
+	sender, err := mail.ParseAddress(setting.MailService.From)
```

Comment:
I think we should put this process in the `modules/setting` while loading email configs? Add a `FromEmail` field in the struct https://github.com/gogits/gogs/blob/master/modules/setting/setting.go#L716 . So we can print error and halt at start time.